src/App.scss

```diff
@@ -2,7 +2,7 @@ iframe {
   display: none;
 }
 
-html {
+/* html {
```

Comment:
The `html` block is commented out. If this is intentional, you can ignore this comment. Otherwise, uncomment it to apply the font-family styles to the entire document.